for-loop.js

```diff
@@ -0,0 +1,57 @@
+//Challenges
+//Logging values with for loops
+//Write a for loop that logs each number from 1 - 20.
+// for (let i = 1; i < 21; i++) {
+//     console.log(i)
+// }
+//Write a for loop that logs the result of each number from 1 - 20 tripled.
+// for (let i = 1; i < 21; i++) {
+//     console.log(i * 3)
+// }
+//Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+// for (let i = 1; i < 21; i += 2) {
```

Comment:
This for loop will console.log the value at each odd index and place the string odd after it. The challenge question was looking to replace each odd value with the string "odd" and keeping the even numbers to display as a number. For instance,
odd
2
odd
4
So maintain the for loop as line 4 and line 8.
`for(let i = 1; i < 21; i++) {`
Recommending using the modulo operator to determine the odd values and conditional statements to show what to display for the even numbers and the odd numbers. Here's an example,
for(let i = 1; i < 21; i++) {
  if(i % 2 !== 0) {
    console.log("odd")
  } else {
    console.log(i)
  }
}